torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -43,7 +43,23 @@ void UnrollPass::handle(Expr* expr) {
 }
 
 namespace {
-Bool* getPredicate(TensorView* tv, std::vector<Val*> inds) {
+Bool* getPredicate(TensorView* tv, std::vector<Val*> inds_) {
```

Comment:
add a brief comment explaining what getPredicate is supposed to do?

Reply:
Start to collect things that need better comments: https://github.com/csarofeen/pytorch/issues/55